docker-compose.yml

```diff
@@ -17,3 +17,12 @@ services:
       - 9990:9990
     depends_on:
       - db
+  jaeger:
+    image: jaegertracing/all-in-one:0.8.0
+    ports:
+      - 8090:16686
+      - 5775:5775/udp
+      - 5778:5778
+      - 6831:6831/udp
+      - 6832:6832/udp
+      - 14268:14268
```

Comment:
Do all the ports need to be published? Is there a minimal set of ports that can be used?

Reply:
Will update to use fewer ports - if other ports are needed please refer to http://jaeger.readthedocs.io/en/latest/getting_started/ which I initially got the port mappings from.